tests/outputs/make-upgrade-canister-proposal.txt

```diff
@@ -15,6 +15,7 @@ Sending message with
           UpgradeSnsControlledCanister = record {
             new_canister_wasm = blob "\00asm\01\00\00\00";
             canister_id = opt principal "pycv5-3jbbb-ccccc-ddddd-cai";
+            canister_upgrade_arg = null;
```

Comment:
The only thing I think that was missing from this PR is a test where this parameter was not null.